Resources/views/MediaAdmin/list.html.twig

```diff
@@ -35,9 +35,11 @@ file that was distributed with this source code.
 {% endmacro %}
 
 {% block list_table %}
+    {% if datagrid.values['category'] is defined and datagrid.values['category']['value'] is defined %}
```

Comment:
twigfiddle? Wtf? There's an app for that! :trollface:

Thought that was an error, but never tried ^^